### ISSUE TITLE
Document lack of integer "Not" operator

### ISSUE THREAD

**Issue by TerrorJack** [WebAssembly/binaryen#1498]
It's weird that both WebAssembly standard and binaryen only provides And/Or/Xor but doesn't provide a "Not" operator. Since "Not" can be easily implemented with Xor, it shall be convenient to simply add NotInt32/NotInt64 to binaryen unary operators.

**Comment by kripken** [comment]
I agree it's a little weird, yeah. Binaryen follows the wasm spec closely, though, so that optimizing is straightforward - we want one Binaryen IR node to correspond to one wasm node, or as close as possible. So adding a node not in wasm, that we need to expand out, has downsides.

**Comment by TerrorJack** [comment]
@kripken I see, if it's not desirable to implement it in C++, at least we can implement it in markdown?

**Comment by kripken** [comment]
markdown?

**Comment by TerrorJack** [comment]
@kripken Write the workaround in docs, for the convenience of later compiler authors

**Comment by kripken** [comment]
Oh, I see. Yeah, that's a good idea I think :)

**Comment by tlively** [comment]
Binaryen doesn't really aim to document how to target WebAssembly, so I think we're ok without mentioning this in the docs (although I wouldn't object if someone opened a PR for it either). I'll go ahead and close this.
